### PR TITLE
Add verified_teachers to javabuilder token

### PR DESCRIPTION
**Pull request by molly-moen**
Add a verified_teachers parameter to the javabuilder token. The parameter is an array of teacher ids. If the current user is a verified teacher, csa pilot teacher or levelbuilder the array will just be their id. If they are not, the array will be a list of all teachers of the user that are verified teachers or csa pilot teachers.

## Links

- jira ticket: [JAVA-457](https://codedotorg.atlassian.net/browse/JAVA-457)

## Testing story
Tested locally that users can still connect to javabuilder and added unit tests that the list of teachers is as expected.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/controllers/javabuilder_sessions_controller.rb [hunk 1]
```diff
@@ -8,9 +8,12 @@ class JavabuilderSessionsController < ApplicationController
 
   PRIVATE_KEY = CDO.javabuilder_private_key
   PASSWORD = CDO.javabuilder_key_password
+  CSA_PILOT = 'csa-pilot'
+  CSA_PILOT_FACILITATORS = 'csa-pilot-facilitators'
```
Comment (jmkulwik) [inline dashboard/app/controllers/javabuilder_sessions_controller.rb:12]: Is there a variable somewhere that is the global source of truth for these? Mostly thinking about how we find all instances of this once the pilot is over.

File: dashboard/test/controllers/javabuilder_sessions_controller_test.rb [hunk 2]
```diff
@@ -138,4 +138,41 @@ class JavabuilderSessionsControllerTest < ActionController::TestCase
...
+    levelbuilder = create :levelbuilder
+    sign_in(levelbuilder)
+    get :get_access_token, params: {channelId: @fake_channel_id, projectVersion: 123, projectUrl: URL, levelId: 261, executionType: 'RUN', miniAppType: 'console'}
+
+    response = JSON.parse(@response.body)
+    token = response['token']
+    decoded_token = JWT.decode(token, @rsa_key_test.public_key, true, {algorithm: 'RS256'})
+    teachers = decoded_token[0]['verified_teachers']
+    assert_equal [levelbuilder.id], teachers
+  end
```
Comment (jmkulwik) [inline dashboard/test/controllers/javabuilder_sessions_controller_test.rb:182]: It'd also be good to check that a non-verified teacher does not get added to the list.

File: dashboard/app/controllers/javabuilder_sessions_controller.rb (hunk 1 above)
Comment (molly-moen) [reply dashboard/app/controllers/javabuilder_sessions_controller.rb:12]: Unfortunately I think the source of truth is in the course file: https://github.com/code-dot-org/code-dot-org/blob/e0794200eddbc8697ace8c940cc0af11ed6b4968/dashboard/config/courses/csa-pilot.course#L19

File: dashboard/test/controllers/javabuilder_sessions_controller_test.rb (hunk 2 above)
Comment (molly-moen) [reply dashboard/test/controllers/javabuilder_sessions_controller_test.rb:182]: done!

**Review by sanchitmalhotra126 — APPROVED** [review]
LGTM!

**Review by jmkulwik — APPROVED** [review]
LGTM

**Pull request merged**